dsfhub/resource_common_test.go

```diff
@@ -0,0 +1,19 @@
+package dsfhub
+
+import (
+	"fmt"
+	"regexp"
+)
+
+func testAccParseResourceAttributeReference(field string) string {
+	var expr string = `dsfhub_[A-Za-z0-9_-].+\.[A-Za-z0-9_-].+` //e.g. dsfhub_cloud_account.my-cloud-account, dsfhub_cloud_account.my-cloud-account.asset_id
+	var ret string
```

Comment:
I know this repo is full of them and we won't go around changing all of those, but when creating new variables maybe we could take a note from Uncle Bob and make their names more readable?

Reply:
Updated the variable names and added a doc comment for the function